src/Elastic.Apm/Api/IExecutionSegment.cs

```diff
@@ -102,6 +102,15 @@ public interface IExecutionSegment
 			Dictionary<string, Label> labels = null
 		);
 
+		/// <summary>
+		/// Captures a log line as an APM error.
+		/// </summary>
+		/// <param name="logOnError"> The logline itself </param>
+		/// <param name="parentId"> ParentId pointing to the parent transaction or span. </param>
+		/// <param name="exception"> Exception which was captured as part of the log. </param>
+		/// <param name="labels">Labels that will be added to the captured error</param>
+		void CaptureLogAsError(ErrorLog logOnError, string parentId = null, Exception exception = null, Dictionary<string, Label> labels = null);
```

Comment:
Thoughts on renaming this to `CaptureErrorLog` ? It'd group the APM error related methods together, and feels like a slightly better description

Reply:
Ok, I'm fine with that. 

Similarly to what I said above, my intention was here also to make clear that here we create APM errors based on the log and this is not log ingestion or anything like that. Just wanted to make sure people don't think that this will show up on the log UI or anything like that - this is pure APM.

But the suggested name is fine, plus we have a comment clearly stating this.